### ISSUE TITLE
Produce correct line/col numbers for on*="tap:" aria rules validation errors

### ISSUE THREAD

**Issue by Gregable** [ampproject/amphtml#1564]
Given a tag such as

`<amp-img on="tap:" src="" width=10 height=10></amp-img>`

Validator produces the errors:

```
foo.html:1:0 MANDATORY_ATTR_MISSING A11Y: Element AMP-IMG#AMP_1 must have "role" attribute due to "tap" action, e.g. role="button".
foo.html:1:0 MANDATORY_ATTR_MISSING A11Y: Element AMP-IMG#AMP_1 must have "tabindex" attribute due to "tap" action.
```

The errors are correct, but the line/col numbers are not. There is a TODO in validator-in-browser.js for fixing this, but tracking it as an explicit issue as well.

cc @powdercloud

**Comment by rudygalfi** [comment]
Looks like we need to move this into the main validator rules as a warning.

**Comment by Gregable** [comment]
Yes, but it should be an error.

**Comment by dvoytenko** [comment]
Should be. But it'd definitely be hard to enforce it now...

**Comment by Gregable** [comment]
We've been emitting these errors for some time, so my guess is that most documents are already compliant. I've asked @honeybadgerdontcare to prototype this and run a regression tests to see how many documents it breaks. Assuming not many, let's make it a real error. If it's significant, we can decide what to do then.

**Comment by Gregable** [comment]
This one is fixed.
